inline_scan.sh

```diff
@@ -523,45 +527,30 @@ get_scan_result_pdf_by_digest() {
 
 save_and_copy_images() {
     # Save all image files to /tmp and copy to created container
-    for image in "${SCAN_IMAGES[@]-}"; do
-        local base_image_name="${image##*/}"
-        echo "Saving ${image} for local analysis"
-        local save_file_name="${base_image_name}.tar"
-        if [[ ! "${image}" =~ [:]+ ]]; then
-            save_file_name="${base_image_name}:latest.tar"
-        fi
-
-        IMAGE_FILES+=("$save_file_name")
+    local base_image_name=$(echo ${FULLTAG} | rev | cut -d '/' -f 1 | rev )
```

Comment:
Taking only the `image-name:tag` part because otherwise it would try to create folders (eg: `docker.io/...`).

Anchore is "smart enough" to find the correct .tar

Reply:
Linux does allow filenames with `:` inside but it's better to avoid it if possible.
Also the `rev | cut | rev` is mind blowing, good job :D

Reply:
I'll try, but I think Anchore needs the full image name or it will fail

Reply:
if Anchore needs it in the name it's ok to leave it like this 👍